apps/user-office-frontend/src/components/user/PeopleTable.tsx

```diff
@@ -255,6 +263,26 @@ const PeopleTable: React.FC<PeopleTableProps> = (props) => {
       onClick: () => setSendUserEmail(true),
     });
 
+  props.setPi &&
+    props.onRemove &&
+    actionArray.push({
+      icon: () => (
+        <Link href="#" underline="hover" variant="subtitle2" component="button">
+          Assign <br /> as PI
+        </Link>
```

Comment:
Why are we using link button as an icon here. We can also use css to style the inline text.